src/Examples/UnitModels/Reactors/equilibrium_reactor_testing.ipynb

```diff
@@ -0,0 +1,1138 @@
+    "- Objective (to define an objective function for optimization)\n",
+    "- TransformationFactory (to apply certain transformations)\n",
+    "- Arc (to connect two unit models)\n",
+    "\n",
+    "For further details on these components, please refer to the pyomo documentation: https://pyomo.readthedocs.io/en/latest/\n",
+    "\n",
+    "From idaes, we will be needing the FlowsheetBlock and the following unit models:\n",
+    "- Mixer\n",
+    "- Compressor\n",
+    "- Heater\n",
+    "- EquilibriumReactor\n",
+    "\n",
+    "We will also be needing some utility tools to put together the flowsheet and calculate the degrees of freedom, tools for model expressions and calling variable values, and built-in functions to define property packages, add unit containers to objects and define our initialization scheme.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyomo.environ import (Constraint,\n",
+    "                           Var,\n",
+    "                           ConcreteModel,\n",
+    "                           Expression,\n",
+    "                           Objective,\n",
+    "                           TransformationFactory,\n",
+    "                           value,\n",
+    "                           units as pyunits)\n",
+    "from pyomo.network import Arc\n",
+    "\n",
+    "from idaes.core import FlowsheetBlock\n",
+    "from idaes.models.properties.modular_properties.base.generic_property import (\n",
+    "        GenericParameterBlock)\n",
+    "from idaes.models.properties.modular_properties.base.generic_reaction import (\n",
+    "        GenericReactionParameterBlock)\n",
+    "from idaes.models.unit_models import (Mixer,\n",
+    "                                      Compressor,\n",
+    "                                      Heater,\n",
+    "                                      EquilibriumReactor)\n",
+    "\n",
+    "from idaes.core.solvers import get_solver\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "from idaes.core.util.initialization import propagate_state"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Importing required thermophysical and reaction packages\n",
+    "\n",
+    "The final step is to import the thermophysical and reaction packages. We will import natural gas properties from an existing IDAES module, and reaction properties from a custom module to describe equilibrium behavior. These configuration dictionaries provide parameter data that we will pass to the Modular Property Framework.\n",
+    "\n",
+    "The reaction package here assumes all reactions reach chemical equilibrium at the given conditions.  \n",
+    "\n",
+    "${K_{eq}^{MSR}} = exp(\\frac {-26830} {T} + 30.114)$, ${K_{eq}^{WGS}} = exp(\\frac {4400} {T} - 4.036)$ with the reactor temperature $T$ in K.  \n",
```

Comment:
It would be better to write `$exp(\frac {-26830} {T} + 30.114)$` as `$\exp\left(\frac {-26830} {T} + 30.114\right)$` so it displays as $\exp\left(\frac {-26830} {T} + 30.114\right)$ instead of $exp(\frac {-26830} {T} + 30.114)$